project/Dependencies.scala

```diff
@@ -56,12 +58,21 @@ object Dependencies {
     "io.monix" %% "newtypes-core" % V.newTypes
   )
 
+  val fastparse = Seq(
+    "com.lihaoyi" %% "fastparse" % V.fastparse
+  )
+
+  val pprint = Seq(
+    "com.lihaoyi" %% "pprint" % V.pprint
```

Comment:
Add the pprint as a dependency to the [`Test` configuration](
https://www.scala-sbt.org/1.x/docs/Library-Dependencies.html#Per-configuration+dependencies)